delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/resolvers/ResolversImpl.scala

```diff
@@ -180,8 +180,10 @@ final class ResolversImpl(
 
 object ResolversImpl {
 
+  final private[resolvers] case class ResolverKey(project: ProjectRef, iri: Iri)
```

Comment:
This seems to be unrelated to the current PR, isn't it?

I would suggest any improvement to the current code to be made in a separate PR. If you want to do this you should do it too for Schemas and Resources

Reply:
I did it because I had to declare the cache key in the serialization bindings

Reply:
this is required to be able to serialize the cache key